src/qt/nametablemodel.cpp

```diff
@@ -270,3 +270,26 @@ NameTableModel::emitDataChanged(int idx)
     //emit
     dataChanged(index(idx, 0), index(idx, columns.length()-1));
 }
+
+QString NameTableModel::renew(const QString &name) const
+{
+    std::string strName = name.toStdString();
+    LogPrintf ("wallet attempting name_update: name=%s\n", strName);
+
+    UniValue params(UniValue::VOBJ);
+    params.pushKV ("name", strName);
+
+    std::string walletURI = ("/wallet/" + walletModel->getWalletName()).toStdString();
+
+    UniValue res;
+    try {
+       res = walletModel->node().executeRpc("name_update", params, walletURI);
+    }
+    catch (const UniValue& e) {
+        UniValue message = find_value( e, "message");
```

Comment:
Nit:  The whitespace is a bit messed up here (in particular the space between `(` and `e`).

Reply:
This style is copied from elsewhere in this file.  I agree it should be fixed.  Should I fix the rest of this file as part of this PR (in a separate commit) while we're thinking of it, or should I do that as a separate PR?

Reply:
Really?  I thought this was just a typo.  I think just fixing the style here is fine for now.